docs/import/center.js

```diff
@@ -20,37 +19,37 @@ function withGeo(feature) {
  * @param {ee.FeatureCollection} features the featureCollection around which you
  * want to orient the map
  * @param {Promise} firebaseAuthPromise
+ * @return {Promise<Array<number>>} Promise that completes with array of bounds
+ * after Firestore write is complete.
  */
 function storeCenter(features, firebaseAuthPromise) {
   const damageWithCoords = ee.FeatureCollection(features.map(withGeo));
   // This is assuming we're not crossing the international date line...
   const outerBounds = ee.List([
-    damageWithCoords.aggregate_max('lat'),
-    damageWithCoords.aggregate_max('lng'),
     damageWithCoords.aggregate_min('lat'),
     damageWithCoords.aggregate_min('lng'),
+    damageWithCoords.aggregate_max('lat'),
+    damageWithCoords.aggregate_max('lng'),
   ]);
-  Promise.all([firebaseAuthPromise, convertEeObjectToPromise(outerBounds)])
-      .then((results) => saveBounds(results[1]))
-      .catch(createError('error finding bounds of map'));
+  return Promise
+      .all([firebaseAuthPromise, convertEeObjectToPromise(outerBounds)])
+      .then((results) => saveBounds(results[1]));
```

Comment:
Why are we not catching errors anymore?

Reply:
Caught errors wouldn't propagate up and be visible to user